Dockerfile

```diff
@@ -135,33 +67,12 @@ RUN set -eux ; \
     rm -rf /tmp/hadoop-${hadoop_version} && \
     rm -f hadoop-${hadoop_version}.tar.gz
 
-# OpenJDK 21
-RUN set -eux ; \
-    ARCH="$(arch)"; \
-    case "${ARCH}" in \
-        x86_64) \
-            url='https://download.java.net/java/GA/jdk21.0.2/f2283984656d49d69e91c558476027ac/13/GPL/openjdk-21.0.2_linux-x64_bin.tar.gz'; \
-            sha256='a2def047a73941e01a73739f92755f86b895811afb1f91243db214cff5bdac3f'; \
-            ;; \
-        aarch64) \
-            url='https://download.java.net/java/GA/jdk21.0.2/f2283984656d49d69e91c558476027ac/13/GPL/openjdk-21.0.2_linux-aarch64_bin.tar.gz'; \
-            sha256='08db1392a48d4eb5ea5315cf8f18b89dbaf36cda663ba882cf03c704c9257ec2'; \
-            ;; \
-        *) echo "Unsupported architecture: ${ARCH}"; exit 1 ;; \
-    esac && \
-    curl -L ${url} -o openjdk.tar.gz && \
-    echo "${sha256} *openjdk.tar.gz" | sha256sum -c - && \
-    tar xzvf openjdk.tar.gz -C /usr/local && \
-    rm -f openjdk.tar.gz
-
-ENV JAVA_HOME=/usr/local/jdk-21.0.2
-# compatibility with Ozone 1.4.0 and earlier compose env.
-RUN mkdir -p /usr/lib/jvm && ln -s $JAVA_HOME /usr/lib/jvm/jre
+ENV JAVA_HOME=/usr/lib/jvm/java-21-openjdk
```

Comment:
Sorry I missed it earlier.  Tried to test the image with Ozone, got:

```
JAVA_HOME /usr/lib/jvm/java-21-openjdk does not exist.
```

It should be `/usr/lib/jvm/jre-21-openjdk`.